### PR TITLE
TRN-296 Confirmaition mail

### PR DESCRIPTION
**Pull request by Kartik9400**
Changed mail template and account setting

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,6 +19,8 @@
         "drupal/core-composer-scaffold": "^9.1",
         "drupal/core-project-message": "^9.1",
         "drupal/core-recommended": "^9.1",
+        "drupal/rules": "3.x-dev@dev",
```
Comment (kkmitra) [inline composer.json]: Why are these modules being installed?

File: config/.htaccess [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Deny all requests from Apache 2.4+.
```
Comment (kkmitra) [inline config/.htaccess]: What is this file is doing in this PR.

**Review by kkmitra — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (Kartik9400) [reply composer.json]: I have installed this module trying to send verification mail to user.

File: config/.htaccess (hunk 2 above)
Comment (Kartik9400) [reply config/.htaccess]: Installed automatically

**Pull request merged**